lib/opensusebasetest.pm

```diff
@@ -312,6 +312,15 @@ sub investigate_yast2_failure {
         "<5>.*Path.*on medium"                  => 'https://trello.com/c/5qTQZKH3/2918-sp2-logs-cleanup',
         "<5>.*Aborting requested by user"       => 'https://trello.com/c/5qTQZKH3/2918-sp2-logs-cleanup',
         "<5>.*Exception.cc"                     => 'https://trello.com/c/5qTQZKH3/2918-sp2-logs-cleanup',
+        # (build97.1) regressions
+        # found https://openqa.suse.de/tests/3646274#step/logs_from_installation_system/412
+        "<3>.*SCR::Dir() failed"                  => 'https://bugzilla.suse.com/show_bug.cgi?id=1158186',
```

Comment:
Please, use format we have in all other cases: bsc#1158186 and not full url.

Reply:
done